components/indicators/IndicatorsHero.tsx

```diff
@@ -1,28 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useRouter } from 'next/router';
-import {
-  Container, Row, Col, Nav, NavItem,
-} from 'reactstrap';
+import { Container, Row, Col, Nav, NavItem } from 'reactstrap';
 import styled from 'styled-components';
 import { readableColor } from 'polished';
 import RichText from 'components/common/RichText';
-import { withTranslation } from '../../common/i18n';
+import { useTranslation, withTranslation } from '../../common/i18n';
```

Comment:
Could change this to absolute path. This is an ancient component from the times we didn't support those yet.